CITATION.cff

```diff
@@ -8,6 +8,10 @@ authors:
     family-names: Andela
     given-names: Bouwe
     orcid: "https://orcid.org/0000-0001-9005-8940"
+  -
+    affiliation: "MetOffice, UK"
+    family-names: Bodas-Salcedo
+    given-names: Alejandro
```

Comment:
It looks like this top section might be for the core team; would it be possible to move this entry to the alphabetical list that starts lower down in this file (around ~L70), please? :)